### ISSUE TITLE
Ubuntu 18.04 breaks DBX by not having python-wnck

### ISSUE THREAD

**Issue by TiZ-EX1** [M7S/dockbarx#68]
So I upgraded from 16.04 to 18.04. There are things about that I both have and have not been looking forward to, and it's been a bit bumpy. The official kernel couldn't even find the root partition!

As far as what *we're* concerned with, the official release of Ubuntu 18.04 just straight up does not have a python-wnck package *at all*. I can find it in launchpad archives and the like, but, for example, the PPA maintained by xuzhen666 is now broken because it's not possible to install the python-wnck dependency. We probably have to load wnck using gir, which it looks like you're already doing on a branch.

Basically, I'm opening up this issue because it's probably time soon to make that *the* way to install DBX. As for me, I've got my own work cut out for me because somewhere between 16.04 and 18.04 they moved the install location of xfce4-panel plugin libs.

**Comment by xuzhen** [comment]
I added the python-wnck package back to my PPA as a temporary solution for 18.04.

**Comment by Falieson** [comment]
```
sudo sh -c "echo 'deb http://cz.archive.ubuntu.com/ubuntu artful main universe' > /etc/apt/sources.list" 
```
then you can run the  apt install command

**Comment by Boxwrapper** [comment]
> ```
> sudo sh -c "echo 'deb http://cz.archive.ubuntu.com/ubuntu artful main universe' > /etc/apt/sources.list" 
> ```
> 
> then you can run the apt install command

Works perfectly!

**Comment by TiZ-EX1** [comment]
I don't know if it's a good idea to intermix main repos of different distro versions *permanently* instead of just grabbing the one specific package.

**Comment by Boxwrapper** [comment]
@TiZ-EX1 Jep, thats true! But until somebody creates a ppa for newer versions, this is pretty much the easiest fix besides compiling and installing manualy. So don‘t considder my answer as a pledge for closing this issue, but rather as information for people that want to get the package installed the „ppa way“.
